### DISCUSSION TITLE
Upload saved/exported file

### DISCUSSION THREAD

**Discussion by SriniJ888** [vueform/builder#29] [Questions] [answered]
Is it possible to upload a saved/work in progress local file (after export), so the user can continue work?  Seems basic enough not to support such a feature.

**Comment by adamberecz** [comment, marked as answer]
Yes, that's possible with our on-prem version (not the online). There you're free to implement saving / loading methods as you see fit, using the builder's API:
- https://builder.vueform.com/docs/saving
- https://builder.vueform.com/docs/loading

Here's how to install the on-prem version in <5 mins:
- https://builder.vueform.com/docs/installation#manual-installation
- https://www.youtube.com/watch?v=0hDF01vUZmY
